bluepysnap/nodes.py

```diff
@@ -239,9 +200,103 @@ def _check_ids(self, node_ids):
             raise BluepySnapError("node ID not found: [%s]" % ",".join(map(str, missing)))
 
     def _check_property(self, prop):
+        """Check if a property exists inside the dataset."""
         if prop not in self.property_names:
             raise BluepySnapError("No such property: '%s'" % prop)
 
+    def _get_node_set(self, node_set_name):
+        """Returns the node set named 'node_set_name'."""
+        if node_set_name not in self._node_sets:
+            raise BluepySnapError("Undefined node set: '%s'" % node_set_name)
+        res = self._node_sets[node_set_name]
+        return None if res == {} else res
```

Comment:
Why is empty set a special case ?
Maybe the `if res == {} then return None` should be in `def _node_sets` ?

Reply:
No this is from an older impl of the node sets where `{}` was interpreted as a None for ids.
I do not allow the `{}` anymore inside the node sets files